### PR TITLE
Suppress `URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD` SpotBugs violations

### PR DESCRIPTION
**Pull request by basil**
Suppresses the following SpotBugs violations:

```
[ERROR] Medium: Unread public/protected field: hudson.model.UpdateCenter$CompleteBatchJob.status [hudson.model.UpdateCenter$CompleteBatchJob] At UpdateCenter.java:[line 2314] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: hudson.search.Search$Item.name [hudson.search.Search$Item] At Search.java:[line 201] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: hudson.security.HudsonPrivateSecurityRealm$SignupInfo.errorMessage [hudson.security.HudsonPrivateSecurityRealm$1] At HudsonPrivateSecurityRealm.java:[line 225] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: hudson.triggers.SCMTrigger$BuildAction.build [hudson.triggers.SCMTrigger$BuildAction] At SCMTrigger.java:[line 426] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.model.ModelObjectWithContextMenu$MenuItem.displayName [jenkins.model.ModelObjectWithContextMenu$MenuItem] At ModelObjectWithContextMenu.java:[line 337] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.model.ModelObjectWithContextMenu$MenuItem.header [jenkins.model.ModelObjectWithContextMenu$ContextMenu] At ModelObjectWithContextMenu.java:[line 141] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.model.ModelObjectWithContextMenu$MenuItem.icon [jenkins.model.ModelObjectWithContextMenu$MenuItem] At ModelObjectWithContextMenu.java:[line 317] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.model.ModelObjectWithContextMenu$MenuItem.post [jenkins.model.ModelObjectWithContextMenu$ContextMenu] At ModelObjectWithContextMenu.java:[line 116] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.model.ModelObjectWithContextMenu$MenuItem.requiresConfirmation [jenkins.model.ModelObjectWithContextMenu$ContextMenu] At ModelObjectWithContextMenu.java:[line 127] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.model.ModelObjectWithContextMenu$MenuItem.subMenu [jenkins.model.Jenkins] At Jenkins.java:[line 4388] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.util.groovy.AbstractGroovyViewModule.f [jenkins.util.groovy.AbstractGroovyViewModule] At AbstractGroovyViewModule.java:[line 30] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.util.groovy.AbstractGroovyViewModule.l [jenkins.util.groovy.AbstractGroovyViewModule] At AbstractGroovyViewModule.java:[line 31] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.util.groovy.AbstractGroovyViewModule.st [jenkins.util.groovy.AbstractGroovyViewModule] At AbstractGroovyViewModule.java:[line 33] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.util.groovy.AbstractGroovyViewModule.t [jenkins.util.groovy.AbstractGroovyViewModule] At AbstractGroovyViewModule.java:[line 32] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.widgets.HistoryPageFilter.hasDownPage [jenkins.widgets.HistoryPageFilter] At HistoryPageFilter.java:[line 62] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.widgets.HistoryPageFilter.hasUpPage [jenkins.widgets.HistoryPageFilter] At HistoryPageFilter.java:[line 61] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.widgets.HistoryPageFilter.nextBuildNumber [jenkins.widgets.HistoryPageFilter] At HistoryPageFilter.java:[line 143] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
[ERROR] Medium: Unread public/protected field: jenkins.widgets.HistoryPageFilter.widget [hudson.widgets.HistoryWidget] At HistoryWidget.java:[line 180] URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD
```

Except for one, these were all false positives due to the field being `@Export`ed for use by Stapler. The one exception was `hudson.triggers.SCMTrigger$BuildAction.build`, a field which is no longer used but was retained for backward compatibility. In either case, there is no further action item so I have suppressed the warnings.

### Proposed changelog entries

N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@StefanSpieker 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Comment by basil** [comment]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Pull request merged**
